examples/tutorial/x/blog/handlers_test.go

```diff
@@ -1,14 +1,166 @@
 package blog
 
 import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
 	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/store"
+	"github.com/iov-one/weave/x"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	weaveCtx weave.Context
+	txs      map[string]weave.Tx
+	handlers map[string]weave.Handler
+	objects  map[string]weave.Persistent
 )
 
+func toWeaveAddress(addr string) weave.Address {
+	d, err := hex.DecodeString(addr)
+	if err != nil {
+		panic(err)
+	}
+
+	return d
```

Comment:
I would validate the address first. So if it is the wrong length/format it will throw an error here.